### DISCUSSION TITLE
Maps creating collaboration

### DISCUSSION THREAD

**Discussion by viT-1** [ihhub/fheroes2#7576] [General]
In [one](https://youtu.be/NFy380A-6OE?t=7984) of youtube streams I had dialog about new maps with your team member.
It's a pitty that for licensing reasons, there is no possibility to add new fan map to your release pack,
because they was created by original Map Editor.
I should recreate my map with fheroes map editor?
Are there any recommendations regarding this? Cloud storing, copyrighting, etc...

**Comment by oleg-derevenetz** [comment]
To be honest, I'm not sure why distribution of new maps created by 3rd parties using the original map editor should violate the 3DO or GoG license. This is separate work and all rights belong to the author of the map, these maps doesn't include any copyrighted material (such as images from the original game, for example). So if any 3rd party person will create a new map and release it with [appropriate GPL-compatible license](https://www.gnu.org/licenses/license-list.en.html#OtherLicenses) (such as CC BY) we should be able to distribute it.

**Comment by ihhub** [comment]
GoG release EULA EULA UBISOFT UK--Jan09.doc file:
```
It is not permitted:
-    To make copies of the Multimedia Product,
...
-    To create or distribute unauthorised levels and/or scenarios, <---- this
-    To decompile, reverse engineer or disassemble the Multimedia Product.
```

**Comment by oleg-derevenetz** [comment]
> To create or distribute unauthorised levels and/or scenarios

So basically you can't use the original map editor at all because by doing this you create "unathorized scenarios" lol. This is kind of absurd.

P.S. What's funny, GoG included many maps made by 3rd-parties in their own distribution.

**Comment by zenseii** [comment]
> P.S. What's funny, GoG included many maps made by 3rd-parties in their own distribution.

We can only assume GOG were given authorization by Ubisoft to include these user-made maps. It is a very absurd license but it leaves us with little other choice.

**Comment by Laserlicht** [comment]
I can't really imagine that the EULA applies here either.

In the files from the CD (dos/windows) I can not find anything. I think a EULA from 2009 can not be applied to backdated releases. If you create the MAPs with the original release, then it's okay, but not with the GOG version?

It's almost like creating/editing images in Photoshop, but then you're not allowed to distribute them...

In fact, the EULA states that you are not allowed to use the map editor. Sounds to me like a copied standard EULA...

Maybe we could do the whole thing by using an extra repo for maps. And that we can download this map pack repo via a button (or at first launch) ingame if you want. So you would not mix it up with this repo.

**Comment by ihhub** [comment]
I am moving this topic into the discussion section as it is definitely not related to the engine.
